pootle/apps/pootle_store/models.py

```diff
@@ -1228,6 +1238,11 @@ def save(self, *args, **kwargs):
 
         self.pootle_path = self.parent.pootle_path + self.name
 
+        # Force validation of "name" field.
+        exclude_fields = list(set([f.name for f in self._meta.get_fields()]) -
+                              set(["name"]))
+        self.clean_fields(exclude=exclude_fields)
```

Comment:
In case someone is asking why only "name" field is being validated, it is because if we use `full_clean` basically all tests fail.